### DISCUSSION TITLE
useEnsResolver can retrieve the ethereum address given the ENS name?

### DISCUSSION THREAD

**Discussion by Javier-Szyfer** [wevm/wagmi#167] [Q&A]
The result from useEnsResolver returns the resolver address but this is not equal to the user ethereum address, is there a possible to achive this with some other wagmi hook if we pass the ens name as a parameter?
thank you

**Comment by tmm** [comment]
Check out [useEnsResolveName](https://wagmi-xyz.vercel.app/docs/hooks/useEnsResolveName)

**Reply by Javier-Szyfer** [reply]
great! did not see this hook until now. thanks guys
